weave-js/src/apollo.ts

```diff
@@ -26,7 +26,9 @@ const authMiddleware = new ApolloLink((operation, forward) => {
   });
 });
 
-export const apolloClient = new ApolloClient({
+// This is a hard-coded client that is used in the old weave.wandb.ai and should
+// be removed with that codebase. Using this is INVALID in any proper wandb app.
+export const deprecatedApolloClientDoNotUse = new ApolloClient({
```

Comment:
I think if you mark this in JSDocs with `@deprecated` it'll strikethrough in editors like VSCode which could help avoid usage.